_includes/commands/table-cmds.md

```diff
@@ -23,7 +23,10 @@
 | mceTablePasteColBefore  | Paste the {{site.productname}} clipboard column before the current row. {{site.requires_5_4v}} |
 | mceTablePasteColAfter   | Paste the {{site.productname}} clipboard column after the current row. {{site.requires_5_4v}} |
 | mceTableDelete          | Deletes the current table.                      |
-| mceInsertTable          | Opens the insert/edit table dialog or inserts a table without using a dialog if additional arguments are provided (see examples below).             |
+| mceTableCellToggleClass | Adds a class to all selected cells that do not have it, or removes the class if all of them have it. {{site.requires_5_9v}}                  |
+| mceTableToggleClass     | Adds a class to all selected tables that do not have it, or removes the class if all of them have it. {{site.requires_5_9v}}                 |
+| mceTableToggleCaption   | Adds a caption to all selected tables that do not have it, or removes the class if all of them have it. {{site.requires_5_9v}}.              |
```

Comment:
These do not apply to multiple tables

Reply:
Yeah, I based the documentation on when we said it would, my bad.